packages/nc-gui/assets/style/style.css

```diff
@@ -362,3 +362,10 @@ html {
 .cursor-pointer{
   cursor: pointer;
 }
+
+
+/* sorting and filter */
+.menu-filter-dropdown {
+  max-height: 500px;
+  overflow-y: scroll;
```

Comment:
Use `overflow-y: auto` to show only when overflows

Reply:
Hi @pranavxc , I have the updated the pull request, according to comment added for Changes request.